lib/classes/commands/GroupCommands.ts

```diff
@@ -291,4 +292,49 @@ export class GroupCommands extends CommandsBase
             });
         });
     }
+
+    async ejectFromGroupBulk(groupID: UUID | string, sendTo: {
+        ejecteeID: UUID | string
+    }[]): Promise<void>
+    {

```

Comment:
I think it would be more efficient and concenient to just have

`sendTo: UUID[] | string[]`

rather than an array of objects containing an ejecteeID.